tests/testcases.py

```diff
@@ -1,109 +1,125 @@
 from datetime import datetime
 from dateutil.relativedelta import relativedelta
 
-
 today = datetime.strptime(datetime.today().strftime("%Y-%m-%d %H:%M:%S"), "%Y-%m-%d %H:%M:%S")
 
 testcases = {
     # Absolute datetime formats
-    "2017.08.03 03:04:05": datetime(year=2017, month=8, day=3, hour=3, minute=4, second=5),
-    "05.05.2017 03:04:10": datetime(year=2017, month=5, day=5, hour=3, minute=4, second=10),
-    "10.01.2022": datetime(year=2022, month=1, day=10),
-    "2023.01.10": datetime(year=2023, month=1, day=10),
-    "03:02:10": datetime(year=today.year, month=today.month, day=today.day, hour=3, minute=2, second=10),
-    "01.01.2023 05:03": datetime(year=2023, month=1, day=1, hour=5, minute=3),
-    "07:16": datetime(year=today.year, month=today.month, day=today.day, hour=7, minute=16),
-    "08:20": datetime(year=today.year, month=today.month, day=today.day, hour=8, minute=20),
+    "absolute_datetime_formats": {
+        "2017.08.03 03:04:05": datetime(year=2017, month=8, day=3, hour=3, minute=4, second=5),
+        "05.05.2017 03:04:10": datetime(year=2017, month=5, day=5, hour=3, minute=4, second=10),
+        "10.01.2022": datetime(year=2022, month=1, day=10),
+        "2023.01.10": datetime(year=2023, month=1, day=10),
+        "03:02:10": datetime(year=today.year, month=today.month, day=today.day, hour=3, minute=2, second=10),
+        "01.01.2023 05:03": datetime(year=2023, month=1, day=1, hour=5, minute=3),
+        "07:16": datetime(year=today.year, month=today.month, day=today.day, hour=7, minute=16),
+        "08:20": datetime(year=today.year, month=today.month, day=today.day, hour=8, minute=20)
+    },
     # Absolute prepositions
-    "second day after christmas": datetime(year=today.year, month=12, day=27),
-    "3rd week of august": datetime(year=today.year, month=8, day=22),
-    "4. week of august": datetime(year=today.year, month=8, day=29),
-    "1st of august": datetime(year=today.year, month=8, day=1),
-    "fifth month of 2021": datetime(year=2021, month=5, day=1),
-    "three days after the fifth of august 2018": datetime(year=2018, month=8, day=8),
-    "second day after august": datetime(year=2022, month=8, day=3),
-    "3 months before the fifth week of august 2020": datetime(year=2020, month=5, day=31),
-    "10 days and 2 hours after 3 months before christmas 2020": datetime(year=2020, month=10, day=5, hour=2),
-    "a day and 3 minutes after 4 months before christmas 2021": datetime(year=2021, month=8, day=26, minute=3),
-    "3 minutes and 4 hours, 2 seconds after new years eve 2000": datetime(year=2000, month=12, day=31, hour=4, minute=3, second=2),
-    "2 days after christmas 2023": datetime(year=2023, month=12, day=27),
-    # Infinity
-    "infinity": None,
-    "inf": None,
```

Comment:
Make another category for general stuff which includes Infinity. I'll add two types later which are placeholders for "Evaluator should return None" and "Evaluator should throw an error"